scripts/cnode-helper-scripts/cntools.sh

```diff
@@ -422,20 +423,22 @@ function main {
                 stake_vk_file="${WALLET_FOLDER}/${wallet_name}/${WALLET_STAKE_VK_FILENAME}"
                 if ! root_prv=$(cardano-address key from-recovery-phrase Shelley <<< ${mnemonic}); then
                   echo && safeDel "${WALLET_FOLDER}/${wallet_name}"
+                  unset mnemonic; unset words
```

Comment:
😄

Reply:
They really only live inside of the subshell created by CNTools anyway but never a bad idea to clear stuff like this as soon as possible.